pennylane/wires.py

```diff
@@ -76,8 +76,11 @@ class Wires(Sequence):
             and hence interpreted as a single wire.
     """
 
-    def __init__(self, wires):
-        self._labels = _process(wires)
+    def __init__(self, wires, _override=False):
```

Comment:
I have never seen "private" kwargs...is there a convention about this?

Reply:
It's to discourage users from using it.